src/routes/oauth/config.ts

```diff
@@ -385,4 +385,25 @@ export const PROVIDERS_CONFIG: Record<
       next();
     },
   },
+  oidc: {
+    grant: {
+      oauth: 2,
+      nonce: true,
+      scope_delimiter: ' ',
+      scope: ['openid', 'profile', 'email'],
+      pkce: process.env.AUTH_PROVIDER_OIDC_PKCE === 'true',
+      authorize_url: `${process.env.AUTH_PROVIDER_OIDC_AUTH_URL}`,
+      access_url: `${process.env.AUTH_PROVIDER_OIDC_TOKEN_URL}`,
+      profile_url: `${process.env.AUTH_PROVIDER_OIDC_USER_INFO_URL}`,
+      client_id: process.env.AUTH_PROVIDER_OIDC_CLIEND_ID,
+      client_secret: process.env.AUTH_PROVIDER_OIDC_CLIENT_SECRET,
+    },
+    profile: ({ profile }) => ({
+      id: profile.sub,
+      email: profile.email,
+      emailVerified: profile.email_verified,
+      displayName: profile.name && profile.nickname,
+      avatarUrl: profile.picture,
```

Comment:
Similar question here. Is this official OIDC profile properties?

````
- email
- email_verified
- name
- nickname
- picture
```

Reply:
Yes, but looks like not all providers returns all of them, that's why I'm trying fallback in some cases (for example with the displayName)

- https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
- https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse
- https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes#standard-claims